code/game/gamemodes/game_mode.dm

```diff
@@ -328,6 +329,7 @@
 	return 0
 
 /datum/game_mode/New()
+	player_draft_log = list()

```

Comment:
This should really be initialized where it's defined, rather than initialized separately in `New`.